pandas/core/internals/blocks.py

```diff
@@ -1531,11 +1531,11 @@ def putmask(self, mask, new) -> list[Block]:
 
         return [self]
 
-    def delete(self, loc) -> Block:
+    def delete(self, loc) -> list[Block]:
```

Comment:
for 2D EAs we probably want to do the same thing as we do for NumpyBlock?

Reply:
Do we have any 2D EAs? I agree in general, but thought all of them were 1D

Reply:
see NDArrayBackedExtensionBlock